client/src/form/EditProfile.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import {Button, Typography, Grid, TextField, Box} from '@material-ui/core';
+import AuthContext from '../state_management/AuthContext';
+import ShListsContext from '../state_management/ShListsContext';
+import axios from 'axios';
+
+const CustomTextField = withStyles({
+    root: {
+      '& .MuiOutlinedInput-root': {
+        '& fieldset': {
+          border: '0',
+        },
+      },
+    },
+  })(TextField);
+
+const useStyles = makeStyles((theme) => ({
+    button: {
+        marginTop: theme.spacing(4),
+        marginBottom: theme.spacing(4)
+    },
+    title: {
+        fontWeight: 'bold'
+    },
+    text: {
+        marginLeft: theme.spacing(1),
+        marginRight: theme.spacing(1),
+    },
+    hidden:{
+		visibility: "hidden",
+    },
+    shown:{
+		visibility: "visible",
+    },
+    box:{
+        backgroundColor: 'white',
+        paddingLeft: theme.spacing(2),
+        paddingRight: theme.spacing(2),
+        marginTop: theme.spacing(4),
+        marginBottom: theme.spacing(4)
+    },
+    input: {
+        display: 'none',
+    }
+}));
+
+export default function BasicTextFields() {
```

Comment:
This is not a very descriptive component name. This component is a form that updates the profile, so the name should reflect that

Reply:
Yikes! I will change it.